src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -270,6 +284,22 @@ private void triggerChangePullBeforeOverride() {
   }
 
 
+  private void triggerOverwriteExistingFiles() {
```

Comment:
Following my comment on the checkbox's action listener... This method's name doesn't accurately express what's happening inside. If we can't find a proper name (because it's too long or too complex) it's often a sign telling us that we need to split the method and separate concerns until we can do so.

This method, and others like this one, handle triggering changes. If you take a look to other `triggerXYZ()` methods, they don't have any conditions inside them. They only iterate their corresponding callbacks and call them. This is another sign telling us that `triggerOverwriteExistingFiles()` should not do anything more than calling callbacks.

Reply:
Makes sense. Will fix it. Removing the condition to the constructor before triggering this method as you requested above makes more sense.